corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -698,7 +698,7 @@ def set_cached_payload_if_necessary(self, fileobj, duration, is_async):
         return None
 
     def delete_cached_payload_if_necessary(self):
-        if self.overwrite_cache and self.restore_payload_path_cache.get_value():
+        if self.overwrite_cache and self.restore_payload_path_cache.exists():
```

Comment:
Made an assumption there that the value can never be a falsy value so it was fine to just check for the presence of the key.